contents/product-engineers/dogfooding.md

```diff
@@ -0,0 +1,137 @@
+
+For feedback is critical to the dogfooding process and to make people comfortable with giving feedback, you need trust.  Your team must trust each other to give and [receive feedback](/handbook/people/feedback). Assume positive intent, acknowledge it, and do with it what you want. This might be uncomfortable, but dogfooding doesn't work without it.
+
+### 2. Implement dogfooding into product development
+
+After creating the culture to support dogfooding, it must be integrated into your process for improving your product. When done right, it can have an impact on multiple stages from ideation to testing to post-release. 
+
+We've found three details are key to getting the most out of dogfooding:
+
+1. **Treat your team like a customer.** [Interview them](/newsletter/talk-to-users), ask them for feedback on what you built, use this to guide your next iteration, and repeat. For the best results, go out of your way to ask for feedback and make your ask specific.
+
+2. **Create tighter feedback loops.** At PostHog, we have [product engineers](/blog/what-is-a-product-engineer) who take full ownership of a product. They both figure out what to build and build it. This means they dogfood and can implement the improvements from dogfooding, iterating rapidly. If you never action feedback, your dogfooding is going to waste.
+
+3. **Separate deployment from release.** Doing this enables you to dogfood a change without releasing it to all your users. Using [feature flags](/feature-flags), you can [test in production](/product-engineers/testing-in-production), fix issues, and ship a polished product to users. 
+
+## The benefits of dogfooding
+
+So after all of this work to get dogfooding right, what do you actually get out of it?
+
+- **Fast feedback.** Getting feedback from your teammates is a lot easier and faster than getting it from users. This guides product development and helps you iterate rapidly.
+
+- **Reveals issues and bugs.** When we dogfood, we almost always discover usability issues or bugs that we can fix before they impact user experience. This helps you ship a polished product.
+
+- **Increases product knowledge.** Our whole team uses PostHog. This helps us fix cross-functional issues, discover use cases, and promote PostHog better.
+
+- **Spirit.** If your product is good, it helps your team become confident in it. If your product is bad, it motivates the team to make it better. As a dramatic example of this, Apple banned typewriters. They were either going to manifest the future of computing or die trying.
+
+![Apple](https://res.cloudinary.com/dmukukwp6/image/upload/apple_fd647f303d.png)
+
+## Avoid falling into the dog bowl (aka the pitfalls of dogfooding)
+
+If you dogfood too much, you might forget what the taste of real food is like. You think you are building what users want, but in reality, are only building you want. To ensure this doesn't happen, here are some pitfalls to avoid: 
```

Comment:
style point but i think the joke on dogfood the process and dogfood for dogs is getting a bit thin. also this is a bit gross?? and it'd be better to get to the point